source/dotnet/Library/CppConsoleApp/main.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "iostream"
+
+using namespace winrt;
+using namespace Windows::Foundation;
+
+// Sample Test App for AdaptiveCards.Template project which is a C#/WinRT component
+int main()
+{
+    init_apartment();
+
+    winrt::hstring input{
```

Comment:
if you already added the "using namespace winrt;" directive, you should be able to remove the "winrt::" prefix from all the spots you're using it in this file